app/frontend/src/components/DateField.tsx

```diff
@@ -0,0 +1,63 @@
+import DateFnsUtils from "@date-io/dayjs";
+import { KeyboardDatePicker, MuiPickersUtilsProvider } from "@material-ui/pickers";
+import { Control, Controller } from "react-hook-form";
+
+interface DateFieldProps {
+  className?: string;
+  control: Control;
+  error: boolean;
+  helperText: any;
+  id: string;
+  inputRef: (ref: any) => void;
+  label: string;
+  name: string;
+}
+
+export default function DateField({
+  className,
+  control,
+  error,
+  helperText,
+  id,
+  inputRef,
+  label,
+  name
+}: DateFieldProps) {
+  return (
+    <MuiPickersUtilsProvider utils={DateFnsUtils}>
+      <Controller
+        control={control}
+        defaultValue={new Date().toISOString().split("T")[0]}
+        inputRef={inputRef}
+        name={name}
+        
+        render={({ onChange, value }) => (
+          <KeyboardDatePicker
+            animateYearScrolling={true}
+            autoOk
+            className={className}
+            clearable
+            disableToolbar
+            error={error}
+            format="DD.MM.YYYY"
+            fullWidth
+            helperText={helperText}
+            id={id}
+            InputLabelProps={{
+              shrink: true,
+            }}
+            label={label}
+            onChange={(date) => {
+              //user might be typing, so check the date is valid before doing conversions
+              date?.isValid()
+                ? onChange(date?.format().split("T")[0])
```

Comment:
What happened when you tried `.toISOString()`?

Reply:
After clicking e.g 18.03.2021 I got different dates. First `console.log()`, when I used `.toISOString()`. Second with `.format()`. The second one was correct. It was connected with timezone. 
`2021-03-17T23:00:00.000Z`
`2021-03-18T00:00:00+01:00`